python/tests/complex_type_tests/test_udt_of_map.py

```diff
@@ -0,0 +1,86 @@
+from tests.aggregate_tests.aggtst_base import TstView, TstTable
+
+
+# Depth 2 tests => User Defined Type of Map Type
+class cmpxtst_udt_of_map_tbl(TstTable):
+    """Define the table used by the user defined type of Map type tests"""
+
+    def __init__(self):
+        self.sql = """CREATE TYPE map_type1 AS(m1_int MAP<VARCHAR, INT> NOT NULL, m2_int MAP<VARCHAR, INT>);
+                      CREATE TYPE map_type2 AS(m1_var MAP<VARCHAR, VARCHAR> NOT NULL, m2_var MAP<VARCHAR, VARCHAR>);
+                      CREATE TABLE udt_of_map_tbl(
+                      id INT,
+                      c1 map_type1,
+                      c2 map_type2 NOT NULL)"""
+        self.data = [
+            {
+                "id": 0,
+                "c1": None,
+                "c2": {"m1_var": {None: "hi", "b": "bye"}, "m2_var": {None: None}},
```

Comment:
if this works it looks like a bug, since the key in map is None

Reply:
please file an issue about this; the resulting test should not compile, or it should fail at runtime.

Reply:
the bug should also say that we don't document what happens if a key in a map is null